src/library/scala/collection/ArrayOps.scala

```diff
@@ -678,10 +678,10 @@ final class ArrayOps[A](private val xs: Array[A]) extends AnyVal {
     *  @return  the index `>= from` of the first element of this array that satisfies the predicate `p`,
     *           or `-1`, if none exists.
     */
-  def indexWhere(f: A => Boolean, from: Int = 0): Int = {
+  def indexWhere(p: A => Boolean, from: Int = 0): Int = {
```

Comment:
Precedent is to use `@deprecatedName`, there are currently just a couple of examples in the library: https://github.com/scala/scala/blob/2.13.x/src/library/scala/collection/mutable/ArrayBuffer.scala#L192

Reply:
Thanks! I didn't realize that I was breaking the contract.